slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/NoOpIncrementalInterchangeV5Exporter.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.slashingprotection.interchange;
+
+import tech.pegasys.web3signer.slashingprotection.interchange.model.Metadata;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.UncheckedIOException;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+
+/** Only export metadata in JSON format without gvr and slashing protection data. */
+public class NoOpIncrementalInterchangeV5Exporter implements IncrementalExporter {
+  private final JsonGenerator jsonGenerator;
+  private static final JsonMapper JSON_MAPPER = new InterchangeJsonProvider().getJsonMapper();
+
+  public NoOpIncrementalInterchangeV5Exporter(final OutputStream outputStream)
+      throws UncheckedIOException {
+    try {
+      jsonGenerator = JSON_MAPPER.getFactory().createGenerator(outputStream);
+      jsonGenerator.writeStartObject();
+      final Metadata metadata = new Metadata("5", null);
```

Comment:
Could reuse the existing FORMAT_VERSION constant here?

Reply:
yes, meant to. Good catch!